includes/preview.php

```diff
@@ -273,5 +287,6 @@ public function wp_footer() {
 	 */
 	public function __construct() {
 		add_action( 'template_redirect', [ $this, 'init' ], 0 );
+		Loading_Inspection_Manager::instance()->register_tests();
```

Comment:
Move to init - after the ensure it's preview.

Reply:
moved to editor.php init

Reply:
Thanks